File: sdks/thunkmetrc-go/wrapper/models/locations_type.go
package models

type LocationsType struct {
    ForHarvests bool `json:"ForHarvests,omitempty"`
    ForPackages bool `json:"ForPackages,omitempty"`
    ForPlantBatches bool `json:"ForPlantBatches,omitempty"`
    ForPlants bool `json:"ForPlants,omitempty"`
    Id int64 `json:"Id,omitempty"`
    Name string `json:"Name,omitempty"`
}

File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/LocationsType.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class LocationsType
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("ForHarvests")]
        public required bool ForHarvests { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("ForPackages")]
        public required bool ForPackages { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("ForPlantBatches")]
        public required bool ForPlantBatches { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("ForPlants")]
        public required bool ForPlants { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Id")]
        public required int Id { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Name")]
        public required string Name { get; set; }
    }
}
